appengine/flexible/tasks/main.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""App Engine app to serve as an endpoint for App Engine queue samples."""
```

Comment:
This is for push queues, yeah? Can we be explicit about that?

Reply:
Yeah, I asked about it but I was told by the PM that the "push queue" terminology is being dropped in favor of "App Engine queue".